back/engines/commercial/admin_api/app/graphql/admin_api/types/idea_type.rb

```diff
@@ -43,6 +43,7 @@ def large_url
     field :upvotes_count, Integer, null: false
     field :downvotes_count, Integer, null: false
     field :comments_count, Integer, null: false
+    field :internal_comments_count, Integer, null: false
```

Comment:
Not entirely sure this is necessary, but working on assumption it doesn't hurt (for now)

Reply:
✅ I added a test into the appropriate graphql spec.